server/src/ArmNagiosNDOUtils.cc

```diff
@@ -588,12 +590,32 @@ void ArmNagiosNDOUtils::getTrigger(void)
 		itemGroupStream >> trigInfo.brief;    // output
 		itemGroupStream >> trigInfo.hostId;   // host_id
 		itemGroupStream >> trigInfo.hostName; // hosts.display_name
+		trigInfo.validity = TRIGGER_VALID;
 		triggerInfoList.push_back(trigInfo);
 	}
+}
+
+void ArmNagiosNDOUtils::getTrigger(void)
```

Comment:
This method and getAllTrigger are very similar and there're many duplicated lines.